src/test/java/org/apache/commons/lang3/ClassUtilsTest.java

```diff
@@ -540,12 +539,11 @@ public void test_getShortCanonicalName_Object() {
         class Named {
             // empty
         }
-        // WARNING: this is fragile, implementation may change, naming is not guaranteed
-        assertEquals("ClassUtilsTest.9", ClassUtils.getShortCanonicalName(new Object() {
+        assertEquals("", ClassUtils.getShortCanonicalName(new Object() {
             // empty
         }, "<null>"));
-        assertEquals("ClassUtilsTest.9Named", ClassUtils.getShortCanonicalName(new Named(), "<null>"));
-        assertEquals("ClassUtilsTest.Inner", ClassUtils.getShortCanonicalName(new Inner(), "<null>"));
+        assertEquals("", ClassUtils.getShortCanonicalName(new Named(), "<null>"));
```

Comment:
This seems obviously no good to me because "Named" is a real concrete class defined in this very file, it's name is not "". What am I missing?

Reply:
@thiyagu-7 
Thank you for the clarification!